src/Compute/Compute/ChangeLog.md

```diff
@@ -21,6 +21,11 @@
 ## Upcoming Release
 * Updated cmdlets with plural nouns to singular, and deprecated plural names.
 
+## Version 1.8.0
```

Comment:
Please just add the new comments to the 'Upcoming Release' section

Reply:
Fixed in 3efe163